test/ecto/query_test.exs

```diff
@@ -13,6 +13,14 @@ defmodule Ecto.QueryTest do
     end
   end
 
+  test "query functions do not require binding" do
+    from(p in "posts") |> limit(1)
+    from(p in "posts") |> order_by([asc: :title])
+    from(p in "posts") |> where(title: "foo")
+    from(p in "posts") |> offset(1)
+    from(p in "posts") |> update(set: [title: "foo"])
```

Comment:
I'm not sure about these tests, but I couldn't find another set of tests that looked like a good place to put them. I used the same style as other tests in this file. Is this sufficient or should I add more robust tests elsewhere? If so, where?